### ISSUE TITLE
For default style featurettes, aria-labelledby should be removed from the "Read More" link

### ISSUE THREAD

**Issue by rebecca-patton** [wvuweb/wvu-design-system#301]
## Steps to reproduce the issue

  1. Add default featurettes to a [page](https://davisonline.wvu.edu/academics/graduate-certificates) and add labels to pull child page data into them.
  1. Run Siteimprove Accessibility Checker extension on page

## Results

Accessibility error 'Visible label and accessible name do not match' is present for 'Read More' links

## To Fix

  1. Remove aria-labelledby attribute on [Line 104](https://github.com/wvuweb/wvu-design-system/blob/master/views/includes/_wvu-featurettes__default.html) (sr-only span meets accessibility needs)
